ui/src/app/utils/workbench-gapi-client.ts

```diff
@@ -8,6 +11,13 @@ interface GoogleBillingAccountInfo {
 export async function getBillingAccountInfo(googleProject: string) {
   return new Promise<GoogleBillingAccountInfo>((resolve) => {
     gapi.load('client', () => {
+      if(isTestAccessTokenActive()) {
```

Comment:
nit: spacing `if (isTestAccess...) {`

surprised linter doesn't catch this one